lib/assessment.js

```diff
@@ -32,13 +42,20 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
+  var newAry = [];
+
+  newAry = array.forEach(callback(currentValue, index, array))
 
+  return newAry;
 };
 
 // Question 5
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
+  if (callback(currentValue, index, array)) {
+    return currentValue;
+  }
```

Comment:
Actually, this would be correct if you put the code inside a `for` loop. As written, though, it will not work.